.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I'm cautiously in favour of it: it doesn't cause many problems for any non-vsc users, especially since it's a hidden file so you often even see it, and it makes life much nicer for vsc users, which I'm expecting that to be most people. I don't have strong views here though.